docs/api/features/index.rst

```diff
@@ -53,6 +49,4 @@ is called which displays the image.
    pcoords
    pca
    manifold
-   importances
-   rfecv
```

Comment:
I know this was a bit of a pain but thanks for sticking with it - I think this is an important move that makes a lot of sense.